tests/copyright_check/copyright_validator.py

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import print_function
+
+import difflib
+
+import os
+import re
+import sys
+import pathlib
+import config
+
+print_logs = config.print_logs
+
+
+def fetch_samplefile():
+    """
+    Method to fetch sample copyright file
+    :return sample
+    """
+    sample = dict()
+    path = os.path.join(config.default_check_dir, config.default_sample_file)
+    ext = pathlib.Path(config.default_sample_file).suffix.split(".")[1]
+    sample_file = open(path, "r")
+    ref = sample_file.read().splitlines()
+    sample_file.close()
+    sample[ext] = ref
+
+    return sample
+
+
+def verify_file(filename, sample, formula):
+    """
+    Method to check if file meets copyright signature expectations
+    :param filename file to check
+    :param sample copyright signature file
+    :param formula regex to verify
+    :return bool
+    """
+    try:
+        f = open(filename, "r")
+    except Exception as exc:
+        print("Unable to open %s: %s" % (filename, exc), file=print_logs)
+        return False
+
+    data = f.read()
+    f.close()
```

Comment:
nit: as before please use context managers to deal with files

```suggestion
    try:
        with open(filename, "r") as f:
            data = f.read()
    except Exception as exc:
        print("Unable to open %s: %s" % (filename, exc), file=print_logs)
        return False
```